examples/basic_voice.py

```diff
@@ -1,9 +1,14 @@
 # SPDX-License-Identifier: MIT
```

Comment:
this example doesn't work

Reply:
https://github.com/DisnakeDev/disnake/pull/716/commits/1f0c71acdc017eb27114d803a365424443975fbd